common/common-iop.mk

```diff
@@ -43,6 +43,12 @@ IOP_LIBS += -lkernel -lgcc
 %.o : %.s
 	$(IOP_AS) $(IOP_ASFLAGS) $< -o $@
 
+%.o : %.lst
+	echo "#include \"irx_imports.h\"" > build-imports.c
```

Comment:
I meant, one globally.

We do something a bit similar with imports on the psp, but we have them named different things - like `kernel-imports` and `dmac-imports` etc.

Would we just use:
```c
// thbase-imports.c
#include <thbase.h>
thbase_IMPORTS
```

Or I guess that's bad since it omits some potential testable imports, like 32 or 31.

Oh well, I guess this comes from the ps2sdk:
https://github.com/ps2dev/ps2sdk/blob/master/iop/Rules.make#L42

Seems like a strange way to do it to me, like many other things in the ps2sdk, but okay.

-[Unknown]